requirements.txt

```diff
@@ -2,7 +2,7 @@ alembic
 Flask
 Flask-WTF
 MarkupSafe
-pygit2
+pygit2==0.22.0
```

Comment:
While I agree with you, it's a hard-one to pin down as it depends on the version of libgit2 that's installed.

That being said, it might be a good place to pin it down actually to show early potential problems :/